pypesto/sampling/metropolis.py

```diff
@@ -32,13 +33,14 @@ def initialize(self, problem: Problem, x0: np.ndarray):
         self.trace_x = [x0]
         self.trace_fval = [self.objective(x0)]
 
-    def sample(self, n_samples: int, beta: float = 1.):
+    def sample(self, n_samples: int, beta: float = 1., hide_bar: bool = False):
```

Comment:
this way, you cannot set the flag via `pypesto.sampling.sample.sample()`, right? I would say make `hide_bar` an argument of the `__init__` method instead.

Reply:
strictly speaking, we cannot change the `sample` function signature, which is only allowed to take the arguments `n_samples` and `beta`.

Reply:
at the moment the progress bar will be displayed by default, may make sense to be able to disable it as well

Reply:
> strictly speaking, we cannot change the `sample` function signature, which is only allowed to take the arguments `n_samples` and `beta`.

Then how would you suggest to do that?

Reply:
@elbaraim I made a suggestion in https://github.com/ICB-DCM/pyPESTO/pull/368, PR to here.